packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -83,8 +83,9 @@ contract OPContractsManager is ISemver, Initializable {
         FaultDisputeGame faultDisputeGame;
         PermissionedDisputeGame permissionedDisputeGame;
         DelayedWETH delayedWETHPermissionedGameProxy;
-        DelayedWETH delayedWETHPermissionlessGameProxy;
     }
+    // TODO: Eventually switch from Permissioned to Permissionless
+    // DelayedWETH delayedWETHPermissionlessGameProxy;
```

Comment:
nit, can we put the TODO and declaration within the struct brackets, just so it's clear where the `DelayedWETH delayedWETHPermissionlessGameProxy` should go?

Reply:
Actually, this is going to be a production contract before we add this back in, and I don't love shipping a production contract with TODOs, since that might feel dirty to customers. We can have comments indicating support for the permissionless game will be added in the future, but let's remove the TODOs from the contract

Reply:
Good call: https://github.com/ethereum-optimism/optimism/pull/12156/commits/9a33062c7d2e941664c0b10802b402ef00ae4430